definitions/ext-maxdb/dashboard.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "E-MAXDB",
```

Comment:
```suggestion
  "name": "EXT-MAXDB",
```

Reply:
Good catch. I'll try to make the dashboard name to be consistent with the entity type.